src/libs/OptionsListUtils.js

```diff
@@ -213,25 +213,30 @@ function isPersonalDetailsReady(personalDetails) {
  * @returns {Array}
  */
 function getParticipantsOptions(report, personalDetails) {
-    const participants = lodashGet(report, 'participantAccountIDs', []);
-    return _.map(getPersonalDetailsForAccountIDs(participants, personalDetails), (details) => ({
-        keyForList: details.login,
-        login: details.login,
-        accountID: details.accountID,
-        text: details.displayName,
-        firstName: lodashGet(details, 'firstName', ''),
-        lastName: lodashGet(details, 'lastName', ''),
-        alternateText: Str.isSMSLogin(details.login || '') ? LocalePhoneNumber.formatPhoneNumber(details.login) : details.login,
-        icons: [
-            {
-                source: UserUtils.getAvatar(details.avatar, details.accountID),
-                name: details.login,
-                type: CONST.ICON_TYPE_AVATAR,
-            },
-        ],
-        payPalMeAddress: lodashGet(details, 'payPalMeAddress', ''),
-        phoneNumber: lodashGet(details, 'phoneNumber', ''),
-    }));
+    const participantsAccountIDs = lodashGet(report, 'participantAccountIDs', []);
+    const participants = getPersonalDetailsForAccountIDs(participantsAccountIDs, personalDetails);
+    return _.map(participants, (details, index) => {
+        const userLogin = Str.removeSMSDomain(details.login || '') || 'Hidden';
```

Comment:
This is not translated..but it isn't translated [here](https://github.com/Expensify/App/blob/332d2c96e1f0b4bbcdd85859868b6d7b3047d76a/src/pages/ReportParticipantsPage.js#L62) either, should I fix this in this PR for both locations?